include/sys/proc.h

```diff
@@ -193,6 +193,8 @@ int do_fork(void (*start)(void *), void *arg, pid_t *cldpidp);
 /*! \brief Set login name associated with current session. */
 int do_setlogin(const char *name);
 
+void get_proc_lists(proc_list_t **ps, proc_list_t **zs);
```

Comment:
I'm in favor of making both list globally available. There's no need to introduce this weird getter.

Reply:
done